src/Symfony/Component/Translation/DataCollectorTranslator.php

```diff
@@ -102,6 +102,8 @@ public function warmUp(string $cacheDir): array
      */
     public function getFallbackLocales(): array
     {
+        trigger_deprecation('symfony/translation', '6.2', 'DataCollectorTranslator::getFallbackLocales() is deprecated. Get the FallbackLocaleProvider instead.');
```

Comment:
```suggestion
        trigger_deprecation('symfony/translation', '6.2', '"%s::getFallbackLocales()" is deprecated, use "%s::getUltimateFallbackLocales()" instead.', DataCollectorTranslator::class, FallbackLocaleProvider::class);
```

Reply:
6.2 must be replaced with 6.4 here and in all other `trigger_deprecation` occurrences.